lib/interpreters/split_clustering.rb

```diff
@@ -691,41 +691,50 @@ def self.split_balanced_kmeans(service_vrp, nb_clusters, options = {}, &block)
       options = defaults.merge(options)
       vrp = service_vrp[:vrp]
       # Split using balanced kmeans
-      if vrp.services.all?{ |service| service&.activity&.point&.location } && nb_clusters > 1
-        cumulated_metrics = Hash.new(0)
+      if vrp.services.all?{ |service| service.activity&.point&.location } && nb_clusters > 0
```

Comment:
`nb_cluster` should never be `<= 0`. I don't know why I left it like that. If it is <= 0 then we did something wrong somewhere and it is a runtime error. I think we can separate this case into a `raise`, what do you think?

Reply:
I was going to suggest to remove nb_clusters > 0 because it is always true.

We can add a raise for more safety. However this number is always defined by vehicles.size so it will normally never be < 0 except if we change whole code.